package.json

```diff
@@ -43,7 +43,7 @@
           "${cwd}"
         ],
         "pattern": {
-          "regexp": "^\\[(.*)\\] (err):([A-Za-z\\s]*)([\\:]*\\s)(\\d+):(\\d+):(.*)$",
```

Comment:
What was the problem matcher issue you found?

This change appears to incorrectly parse the error position

### before
![Screen Shot 2023-03-13 at 4 45 49 PM](https://user-images.githubusercontent.com/85081687/224856530-efd3da8e-7c71-4cb4-b27d-6932fdc4dbfc.png)

### after

![Screen Shot 2023-03-13 at 4 39 16 PM](https://user-images.githubusercontent.com/85081687/224856547-cdb2d72a-3efd-4ccd-8c5b-6016a09eb715.png)

Reply:
Reverted my change, me and regex do not get along at all.